[
    {
        "question": "I am designing a vehicle tracking program with Deepstream SDK 6.0.1. I want to use YoloV8 as my inference model but require the cfg and weights files for that model. When you calibrate your inference model, you can specify which inference model you want to use, by specifying the location of the relevant cfg, and weights files. It is quite easy to find the pre-trained cfg files and weights of Yolov3, but I have been trying to find the weights and cfg files for other Yolo models but could not find any...I have tried to follow the steps from this website: https://wiki.seeedstudio.com/YOLOv8-DeepStream-TRT-Jetson/ , but I receive the error message 'Illegal instruction (code dumped)' when I try to use the pytorch and torchvision packages (I am using the correct versions, and system architecturs of these packages). I then implemented the steps described on the website on my personal computer (windows 11 OS), and got it running, but the python script that they specify on the website (gen_wts_yoloV8.py), is not located in the DeepsteamYolo file that had to be downloaded. The other python script that is located in the file (\"export_yoloV8.py\"), exports a onnx file, which is also not exactly what I want (except if someone know if this can be converted to a cfg and weight files). Does someone know where I could find cfg files and weights for Yolov8, or if it is possible to generate these files? I am programming on a Jetson nano, with Ubuntu 18.04 and have installed the Jetpack 4.6.1 SDK on the SD.",
        "answers": [],
        "votes": []
    },
    {
        "question": "I reboot the nvidia Jetson, after reboot I got message Starting Login Service Started D-Bus system message Bus How can I set up the jetson now fully from initial or how can I debug this error? I am expecting to reboot completely from the initial point, and install new jetpack from the start.",
        "answers": [],
        "votes": []
    },
    {
        "question": "Is there a way to find the version of the currently installed JetPack on my NVIDIA Jetson Xavier AGX kit?",
        "answers": [
            [
                "To get the JetPack version, architecture and dependencies, sudo apt-cache show nvidia-jetpack #Package: nvidia-jetpack #Version: 4.4.1-b50 #Architecture: arm64 #Maintainer: NVIDIA Corporation #Installed-Size: 194 #Depends: nvidia-cuda (= 4.4.1-b50), nvidia-opencv (= 4.4.1-b50), nvidia-cudnn8 (= 4.4.1-b50) For the version specifically, sudo apt-cache show nvidia-jetpack | grep \"Version\" #Version: 4.4.1-b50"
            ],
            [
                "git clone https://github.com/jetsonhacks/jetsonUtilities.git cd jetsonUtilities python jetsonInfo.py Output: NVIDIA Jetson Nano (Developer Kit Version) L4T 32.5.1 [ JetPack 4.5.1 ] Ubuntu 18.04.5 LTS Kernel Version: 4.9.201-tegra CUDA 10.2.89 CUDA Architecture: 5.3 OpenCV version: 3.4.17-dev OpenCV Cuda: YES CUDNN: 8.0.0.180 TensorRT: 7.1.3.0 Vision Works: 1.6.0.501 VPI: ii libnvvpi1 1.0.15 arm64 NVIDIA Vision Programming Interface library Vulcan: 1.2.70"
            ]
        ],
        "votes": [
            12.0000001,
            3.0000001
        ]
    },
    {
        "question": "I'm trying to run this repository with TX2 but I stuck in some errors. The requirement of this repository is flahsing JetPack 4.2 but I couldn't flash it because there's no JetPack 4.2. Then, I've installed JetPack 4.5 on this webpage, not JetPack 4.2. After that, I tried to install wheel file but there are some errors with below codes. sudo apt update sudo apt upgrade sudo apt-get install python-setuptools sudo apt-get install python-pip cd ~/Downloads sudo pip install tensorflow-1.12.1-cp27-cp27mu-linux_aarch64.whl The directory '/home/nvidia/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag. The directory '/home/nvidia/.cache/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag. Processing ./tensorflow-1.12.1-cp27-cp27mu-linux_aarch64.whl Requirement already satisfied: astor&gt;=0.6.0 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: termcolor&gt;=1.1.0 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: numpy&gt;=1.13.3 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: tensorboard&lt;1.13.0,&gt;=1.12.0 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: six&gt;=1.10.0 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: backports.weakref&gt;=1.0rc1 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: absl-py&gt;=0.1.6 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: wheel in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: gast&gt;=0.2.0 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: keras-preprocessing&gt;=1.0.5 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: protobuf&gt;=3.6.1 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: enum34&gt;=1.1.6 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Collecting keras-applications&gt;=1.0.6 (from tensorflow==1.12.1) Collecting mock&gt;=2.0.0 (from tensorflow==1.12.1) Downloading https://files.pythonhosted.org/packages/05/d2/f94e68be6b17f46d2c353564da56e6fb89ef09faeeff3313a046cb810ca9/mock-3.0.5-py2.py3-none-any.whl Requirement already satisfied: grpcio&gt;=1.8.6 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorflow==1.12.1) Requirement already satisfied: markdown&gt;=2.6.8 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorboard&lt;1.13.0,&gt;=1.12.0-&gt;tensorflow==1.12.1) Requirement already satisfied: werkzeug&gt;=0.11.10 in /home/nvidia/.local/lib/python2.7/site-packages (from tensorboard&lt;1.13.0,&gt;=1.12.0-&gt;tensorflow==1.12.1) Requirement already satisfied: futures&gt;=3.1.1; python_version &lt; \"3\" in /home/nvidia/.local/lib/python2.7/site-packages (from tensorboard&lt;1.13.0,&gt;=1.12.0-&gt;tensorflow==1.12.1) Collecting h5py (from keras-applications&gt;=1.0.6-&gt;tensorflow==1.12.1) Downloading https://files.pythonhosted.org/packages/5f/97/a58afbcf40e8abecededd9512978b4e4915374e5b80049af082f49cebe9a/h5py-2.10.0.tar.gz (301kB) 100% |\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588\u2588| 307kB 1.9MB/s Requirement already satisfied: funcsigs&gt;=1; python_version &lt; \"3.3\" in /usr/lib/python2.7/dist-packages (from mock&gt;=2.0.0-&gt;tensorflow==1.12.1) Requirement already satisfied: setuptools&gt;=36 in /home/nvidia/.local/lib/python2.7/site-packages (from markdown&gt;=2.6.8-&gt;tensorboard&lt;1.13.0,&gt;=1.12.0-&gt;tensorflow==1.12.1) Installing collected packages: h5py, keras-applications, mock, tensorflow Running setup.py install for h5py ... error Complete output from command /usr/bin/python -u -c \"import setuptools, tokenize;__file__='/tmp/pip-build-6EHebF/h5py/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\\r\\n', '\\n');f.close();exec(compile(code, __file__, 'exec'))\" install --record /tmp/pip-4T2QcD-record/install-record.txt --single-version-externally-managed --compile: running install running build running build_py creating build creating build/lib.linux-aarch64-2.7 creating build/lib.linux-aarch64-2.7/h5py copying h5py/ipy_completer.py -&gt; build/lib.linux-aarch64-2.7/h5py copying h5py/version.py -&gt; build/lib.linux-aarch64-2.7/h5py copying h5py/__init__.py -&gt; build/lib.linux-aarch64-2.7/h5py copying h5py/h5py_warnings.py -&gt; build/lib.linux-aarch64-2.7/h5py copying h5py/highlevel.py -&gt; build/lib.linux-aarch64-2.7/h5py creating build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/vds.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/group.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/selections.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/dataset.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/filters.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/selections2.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/__init__.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/dims.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/files.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/attrs.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/compat.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/datatype.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl copying h5py/_hl/base.py -&gt; build/lib.linux-aarch64-2.7/h5py/_hl creating build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_filters.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_file2.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_attrs.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_h5t.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/common.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_slicing.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_h5pl.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/__init__.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_attrs_data.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_attribute_create.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_h5d_direct_chunk.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_dataset_getitem.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_h5f.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_group.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_h5p.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_selections.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_datatype.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_dimension_scales.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_file.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_dataset.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_base.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_deprecation.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_file_image.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_dims_dimensionproxy.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_h5.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_objects.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_threads.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_completions.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_dtype.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests copying h5py/tests/test_dataset_swmr.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests creating build/lib.linux-aarch64-2.7/h5py/tests/test_vds copying h5py/tests/test_vds/test_virtual_source.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests/test_vds copying h5py/tests/test_vds/__init__.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests/test_vds copying h5py/tests/test_vds/test_highlevel_vds.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests/test_vds copying h5py/tests/test_vds/test_lowlevel_vds.py -&gt; build/lib.linux-aarch64-2.7/h5py/tests/test_vds running build_ext Traceback (most recent call last): File \"&lt;string&gt;\", line 1, in &lt;module&gt; File \"/tmp/pip-build-6EHebF/h5py/setup.py\", line 159, in &lt;module&gt; cmdclass = CMDCLASS, File \"/home/nvidia/.local/lib/python2.7/site-packages/setuptools/__init__.py\", line 162, in setup return distutils.core.setup(**attrs) File \"/usr/lib/python2.7/distutils/core.py\", line 151, in setup dist.run_commands() File \"/usr/lib/python2.7/distutils/dist.py\", line 953, in run_commands self.run_command(cmd) File \"/usr/lib/python2.7/distutils/dist.py\", line 972, in run_command cmd_obj.run() File \"/home/nvidia/.local/lib/python2.7/site-packages/setuptools/command/install.py\", line 61, in run return orig.install.run(self) File \"/usr/lib/python2.7/distutils/command/install.py\", line 601, in run self.run_command('build') File \"/usr/lib/python2.7/distutils/cmd.py\", line 326, in run_command self.distribution.run_command(command) File \"/usr/lib/python2.7/distutils/dist.py\", line 972, in run_command cmd_obj.run() File \"/usr/lib/python2.7/distutils/command/build.py\", line 128, in run self.run_command(cmd_name) File \"/usr/lib/python2.7/distutils/cmd.py\", line 326, in run_command self.distribution.run_command(command) File \"/usr/lib/python2.7/distutils/dist.py\", line 972, in run_command cmd_obj.run() File \"setup_build.py\", line 166, in run config.run() File \"setup_configure.py\", line 160, in run self.hdf5_version = autodetect_version(self.hdf5) File \"setup_configure.py\", line 196, in autodetect_version import pkgconfig File \"/tmp/pip-build-6EHebF/h5py/.eggs/pkgconfig-1.5.3-py2.7.egg/pkgconfig/__init__.py\", line 1, in &lt;module&gt; from .pkgconfig import * File \"/tmp/pip-build-6EHebF/h5py/.eggs/pkgconfig-1.5.3-py2.7.egg/pkgconfig/pkgconfig.py\", line 281 flags = _query(packages, *os_opts, *_build_options(option, static=static)) ^ SyntaxError: invalid syntax ---------------------------------------- Command \"/usr/bin/python -u -c \"import setuptools, tokenize;__file__='/tmp/pip-build-6EHebF/h5py/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\\r\\n', '\\n');f.close();exec(compile(code, __file__, 'exec'))\" install --record /tmp/pip-4T2QcD-record/install-record.txt --single-version-externally-managed --compile\" failed with error code 1 in /tmp/pip-build-6EHebF/h5py/ Please help me and feel free to ask anything about it. Thank you for reading it.",
        "answers": [],
        "votes": []
    }
]